src/main/java/hbp/mip/configurations/SecurityConfiguration.java

```diff
@@ -60,7 +60,15 @@ public ClientRegistrationRepository clientRegistrationRepository() {
     SecurityFilterChain clientSecurityFilterChain(HttpSecurity http, ClientRegistrationRepository clientRegistrationRepo) throws Exception {
         if (authenticationEnabled) {
             http.authorizeHttpRequests(auth -> auth
-                    .requestMatchers("/login/**", "/oauth2/**", "/actuator/**").permitAll()
+                    .requestMatchers(
+                            "/login/**",
+                            "/oauth2/**",
+                            "/actuator/**",
+                            "/v3/api-docs",
+                            "/v3/api-docs/**",
```

Comment:
Why do we need to expose that?

Reply:
This will provide the swagger template in a json format most have it exposed.